docs/DeveloperGuide.adoc

```diff
@@ -334,30 +458,141 @@ image::UndoRedoActivityDiagram.png[width="650"]
+** Cons: User might think that undo also applies when the list is modified (undoing filtering for example),  only to realize that it does not do that, after executing `undo`.
 * **Alternative 2:** Include all commands.
-** Pros: Might be more intuitive for the user.
-** Cons: User have no way of skipping such commands if he or she just want to reset the state of the address * book and not the view.
+** Pros: PTMan might be more intuitive for the user.
+** Cons: User has no way of skipping such commands if he or she just want to reset the state of PTMan and not the view.
 **Additional Info:** See our discussion  https://github.com/se-edu/addressbook-level4/issues/390#issuecomment-298936672[here].
 
-
 ===== Aspect: Data structure to support the undo/redo commands
 
 * **Alternative 1 (current choice):** Use separate stack for undo and redo
-** Pros: Easy to understand for new Computer Science student undergraduates to understand, who are likely to be * the new incoming developers of our project.
-** Cons: Logic is duplicated twice. For example, when a new command is executed, we must remember to update * both `HistoryManager` and `UndoRedoStack`.
+** Pros: Easy to understand for new Computer Science undergraduates, who are likely to be the new incoming developers of our project.
+** Cons: Logic is duplicated twice. For example, when a new command is executed, we must remember to update  both `HistoryManager` and `UndoRedoStack`.
 * **Alternative 2:** Use `HistoryManager` for undo/redo
 ** Pros: We do not need to maintain a separate stack, and just reuse what is already in the codebase.
-** Cons: Requires dealing with commands that have already been undone: We must remember to skip these commands. Violates Single Responsibility Principle and Separation of Concerns as `HistoryManager` now needs to do two * different things.
+** Cons: Requires dealing with commands that have already been undone: We must remember to skip these commands. Violates Single Responsibility Principle and Separation of Concerns as `HistoryManager` now needs to do two different things.
 // end::undoredo[]
 
+=== Outlet Feature
+
+==== Reason for implementation
+PTMan is designed for outlets such as bubble tea shops and fast food resturants which require a large number of part-time workers.
+This outlet feature stores basic outlet information such as outlet name, operating hours, contact number, admin password and timetable for workers.
+Such details are essential to facilitate other functions such as the access control system and the timetable displayed in the GUI.
+
+==== Current implementation
+
+===== Model component
+
+Currently, the `OutletInformation` object initialized in the `PartTimeManager` is created by calling its default constructor:
+----
+OutletInformation()
+----
+This default constructor initializes the five fields of `OutletInformation` to respective default values:
+----
+    public OutletInformation() {
```

Comment:
Please use code snippets minimally.
Reason: it is difficult to maintain these in the document, particularly if the implementation changes.